src/main/java/com/enonic/xp/app/welcome/market/GetMarketConfigBean.java

```diff
@@ -0,0 +1,28 @@
+package com.enonic.xp.app.welcome.market;
+
+import java.util.function.Supplier;
+
+import com.enonic.xp.script.bean.BeanContext;
+import com.enonic.xp.script.bean.ScriptBean;
+
+public final class GetMarketConfigBean
```

Comment:
This was not an overkill, because `app.config` gives you `com.enonic.xp.app.welcome.cfg` file whereas bean gives `com.enonic.xp.market.cfg` file, where we want to have market config